.github/workflows/content-origin-request.yml

```diff
@@ -0,0 +1,62 @@
+# This starts up a simulator that tries to do what our Lambda@Edge does.
+
+name: content-origin-request
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - deployer/aws-lambda/**
+      - libs/**
+      - .github/workflows/content-origin-request.yml
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup Node.js environment
+        uses: actions/setup-node@v2.1.5
+        with:
+          node-version: "12"
+
+      - name: Cache node_modules
+        uses: actions/cache@v2.1.4
+        id: cached-node_modules
+        with:
+          path: |
+            deployer/aws-lambda/content-origin-request/node_modules
```

Comment:
The safest way to resolve this is to change the line below to `hashFiles('**/yarn.lock')`. Yes, there'll be times that `deployer/aws-lambda/content-origin-response/yarn.lock` changed and you have to yarn install again.

Reply:
I also made the cache key depend on `libs/**/*.js`.

Reply:
See the line below where it creates the `key:`

Reply:
That's a great idea, love it! However, I think we should go back to using `hashFiles('deployer/aws-lambda/content-origin-request/yarn.lock')` for that part of the key instead of `hashFiles('deployer/**/yarn.lock')`, since we're just caching `deployer/aws-lambda/content-origin-request/node_modules`. We don't want the cache key to depend on the `yarn.lock` of all of the other lambda functions, just the files under `libs` and `deployer/aws-lambda/content-origin-request/yarn.lock`. See my suggestion below.